junit-commons/src/main/java/org/junit/gen5/commons/util/ObjectUtils.java

```diff
@@ -29,4 +31,8 @@ public static String nullSafeToString(Class<?>... classes) {
 		return stream(classes).map(Class::getName).collect(joining(", "));
 	}
 
+	public static void verifyNonNull(Object obj, String message) {
+		if (obj == null)
+			throw new IllegalArgumentException(message);
+	}
```

Comment:
This is exactly the same as `Preconditions.notNull()`. Please remove this method again and use the existing one instead. 😉